### DISCUSSION TITLE
[开源计划] v5.u.r-dev

### DISCUSSION THREAD

**Discussion by QIN2DIM** [QIN2DIM/V2RayCloudSpider#52] [吹水]
## Project 

[查看工程进度](https://github.com/QIN2DIM/V2RayCloudSpider/projects)

## PaneL v5.4.r-dev

> 2~4 工作模式的转换会强制共享本机算力，分担群体“上链”的资源消耗。（因发行工程量较大暂时无法开源）

- [ ] 加入 `proxypool 模式`，允许 Panel 客户端拉取开源订阅，一键打开 Clash 并下载配置；
- [ ] 试运行 `优选IP模块`。Panel 将具备「IP优选」以及「CDN 复用」的能力（仅 Windows 可用）。此时终端代理将享有极为夸张的代理加速效果；
- [ ] 试运行 `园丁系统`。Panel 将具备基于本机环境采集、优选开源订阅节点的能力。此时 Panel 本体将切换工作模式，终端程序可获取优质可靠的代理节点；
- [ ] 试运行 `破冰船永久订阅系统`。Panel 将具备从服务堆拉取永久订阅的能力。此时 Panel 本体将切换工作模式，具备一定程度的流量反射能力，根据不同的使用场景实现“坏点自动切换”“忙时分流”“加速访问”“代理维护”等智能驻台功能。

## UtiLs v5.u.r-dev.x

`armour`是一系列针对目前常见的sspanel验证方案的对抗组件。包含如下模块：

- [x] [ArmourGeeTest](https://github.com/QIN2DIM/armour-geetest)
  - [x] v2 滑动验证
  - [x] v3 滑动验证
  - ~~文字点选验证~~
  - ~~图像水平验证~~
- [ ] 邮箱验证
- [ ] Google reCAPTHCA 人机验证

**Comment by wanghaisheng** [comment]
nb  jiayou

**Comment by QIN2DIM** [comment]
> nb jiayou

阿哲，大佬言重了...

**Comment by marcyquan** [comment]


**Reply by QIN2DIM** [reply]
🤷‍♂️？
